src/Jaeger/Jaeger.csproj

```diff
@@ -11,7 +11,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="2.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="2.0.0" />
```

Comment:
Why are you using `2.0.0` for `Microsoft.Extensions.Logging.Abstractions` and `2.0.2` for `Microsoft.Extensions.Logging`? `Microsoft.Extensions.Logging` has it's dependency set to `Microsoft.Extensions.Logging.Abstractions` version `2.0.2`, so we should keep those aligned.

Reply:
Well, it was intentional because I thought that using 2.0.0 would make it easier for people who aren't on the latest patch version yet to use this but as people should move to the latest patch release asap anyway, there's probably not much point in that.

It now targets 2.0.2 - which is the latest patch release.